packages/components/app/styles/components/form/select.scss

```diff
@@ -67,3 +67,35 @@
     }
   }
 }
+
+// MULTIPLE/SIZE
+
+.hds-form-select,
+.hds-form-select:focus {
+  &[multiple],
+  &[size]{
+    background: none;
+
+    option {
+      margin: 2px auto;
+
+      &:hover {
+        color: var(--token-color-foreground-action);
+      }
+      &:disabled {
+        color: var(--token-color-foreground-disabled);
+      }
+      &:checked {
+        border-radius: 3px;
```

Comment:
[super-giga-nitpick] for me this goes on line 80, under the `option` selector.

Reply:
right – will move it there